Headlines/src/ScreenSwitcher/ScreenSwitcherViewController.swift

```diff
@@ -0,0 +1,92 @@
+//
+//  ScreenSwitcherViewController.swift
+//  Canillitapp
+//
+//  Created by Ezequiel Becerra on 11/6/17.
+//  Copyright © 2017 Ezequiel Becerra. All rights reserved.
+//
+
+import UIKit
+
+class ScreenSwitcherViewController: UIViewController {
+
+    public var initialViewController: String?
+    
+    public var currentViewController: UIViewController?
+    
+    override var preferredStatusBarStyle: UIStatusBarStyle {
+        get {
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/Canillitapp/headlines-iOS/blob/6f1c7836a2c0d74d84b1a9f67b523ee54a5b5f94/Headlines/src/ScreenSwitcher/ScreenSwitcherViewController.swift#L18"></span>Computed read-only properties should avoid using the get keyword.</td>
    </tr>
  </tbody>
</table>